sample-app/app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,112 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+/* mixins, variables, etc. */
+$gray-medium-light: #eaeaea;
+$light-black:#333333;
+/* universal */
+body {
+  padding-top: 60px;
+}
+section {
+  overflow: auto;
+}
+textarea {
+  resize: vertical;
+}
+.center {
+  text-align: center;
+h1 {
+  margin-bottom: 10px;
+}
+}
+/* typography */
+h1, h2, h3, h4, h5, h6 {
+  line-height: 1;
+}
+h1 {
+  font-size: 3em;
+  letter-spacing: -2px;
+  margin-bottom: 30px;
+  text-align: center;
+}
+h2 {
+  font-size: 1.2em;
+  letter-spacing: -1px;
+  margin-bottom: 30px;
+  text-align: center;
+  font-weight: normal;
+  color: $gray-light;
+}
+p {
+  font-size: 1.1em;
+  line-height: 1.7em;
+}
+/* header */
+#logo {
+  float: left;
+  margin-right: 10px;
+  font-size: 1.7em;
+  color: white;
+  text-transform: uppercase;
+  letter-spacing: -1px;
+  padding-top: 9px;
+  font-weight: bold;
+
+  &:hover {
+    color: white;
+    text-decoration: none;
+  }
+}
+.navbar-right{
+  .lang{
+    padding-top: 15px;
+    padding-bottom: 15px;
+    color: $gray-light;
+    ul{
+      display: none;
+    }
+    &:hover{
+      ul{
+        display:block;
+        position: absolute;
+        left: -40px;
+        bottom: -80px;
+        li{
+          background-color: $light-black;
+          list-style-type: none;
+          padding: 10px 15px;
+          a{
```

```suggestion
          a {
```
có dấu cách trước { e nhé